app/uk/gov/hmrc/serviceconfigs/model/model.scala

```diff
@@ -25,11 +25,15 @@ case class RepoName(asString: String) extends AnyVal
 
 case class FileName(asString: String) extends AnyVal
 
-case class ServiceName(asString: String) extends AnyVal
+case class ServiceName(asString: String) extends AnyVal {
+  def trimmed: ServiceName = ServiceName(asString.trim)
```

Comment:
It feels like an overreach to override every String function as required. A `modify(a => a)` function on all value objects would be preferrable.

However, the idea of a `ServiceName` type is that it can be trusted to be a ServiceName - not requiring defensive processing - I'd prefer to move this to Query/PathParam binders before populating a ServiceName (if required at all - this is usually handled in the frontend - if there's still a space, then there's no match)